elyra/__init__.py

```diff
@@ -47,6 +48,7 @@ def _load_jupyter_server_extension(nb_server_app):
         (url_path_join(web_app.settings['base_url'], r'/elyra/namespace'), NamespaceHandler),
         (url_path_join(web_app.settings['base_url'], r'/elyra/pipeline/schedule'), PipelineSchedulerHandler),
         (url_path_join(web_app.settings['base_url'], r'/elyra/pipeline/export'), PipelineExportHandler),
+        (url_path_join(web_app.settings['base_url'], r'/elyra/pipeline/operationparser'), OperationParserHandler),
```

Comment:
This needs to specify the path, with a corresponding regex. e.g., `/elyra/pipeline/operationparser/{path}`

I'm also thinking we might want to expose other operation-based functionality and the following might be more amenable to growth: `/elyra/operation/{path}/properties` since we're really just gather properties (envs for now) via this endpoint.